buildSrc/build.gradle

```diff
@@ -168,6 +168,9 @@ if (project != rootProject) {
   forbiddenApisTest.enabled = false
   jarHell.enabled = false
   thirdPartyAudit.enabled = false
+  if (Boolean.parseBoolean(System.getProperty("tests.fips.enabled"))){
```

Comment:
Looking at this again, are we determining whether we are in a fips JVM _only_ by looking at this system property? If so, we should refactor how we set the corresponding `BuildParams` property and use that instead. As I remember, we used to involve a script command using the runtime java home to make this determination but at some point that was removed?

Reply:
You're absolutely right. Before #48378 we would check the SecurityProvider that was in use in runtime and determine if we're running in a FIPS 140 JVM. Nowadays, we explicitly control this with the system property so there is no need to defer to runtime and we can handle everything on configuration. Thus, I followed your advice and removed the `BuildParams.inFipsJvm` and the `globalInfo.ready()` blocks where applicable. Tagging you for another review based on that